_test/const12.go

```diff
@@ -0,0 +1,17 @@
+package main
```

Comment:
I think that instead of this test, we should rather add a simple unit test for isNatural that checks that it returns true for a typed const as well. I can try to do it if you want?